worlds/subnautica/docs/setup_en.md

```diff
@@ -18,20 +18,26 @@
 
```

Comment:
Could you edit
> - Subnautica from: [Subnautica Steam Store Page](https://store.steampowered.com/app/264710/Subnautica/)

to add something like " or other game stores"? The mods work with Subnautica from the Epic Game Store (where Subnautica has been temporarily given out for free in the past). If it helps, the verbiage from QModManager4 is "Run the game from the Steam Library (or other game store library)"